src/main/java/com/denizenscript/denizencore/objects/core/ElementTag.java

```diff
@@ -1469,10 +1468,7 @@ else if (elem.equals("false")) {
         // Returns the value in sentence case (the first letter capitalized, the rest lowercase).
         // -->
         tagProcessor.registerStaticTag(ElementTag.class, "to_sentence_case", (attribute, object) -> {
-            if (object.element.length() == 0) {
-                return new ElementTag("");
-            }
-            return new ElementTag(Character.toUpperCase(object.element.charAt(0)) + object.element.substring(1).toLowerCase());
+            return new ElementTag(object.element.isEmpty() ? "" : Character.toUpperCase(object.element.charAt(0)) + object.element.substring(1).toLowerCase());

```

Comment:
Not sure if I see the benefit in these sorts of changes? For simpler lines sure but this is already pretty cluttered to look at - there's nothing wrong with separating an early return (and also more consistent with the rest of the tags here).